about.html

```diff
@@ -0,0 +1,146 @@
+          “Meet the learning style or needs of students.
+          Improve the efficiency and effectiveness.
+          Improve user-accessibility and time”
+        </h1>
+      </div>
+      <div class="about-ideal">
+        E-Learning is the learning facilitated and supported by Information Communication technologies
+        (ICT) to enable people to learn anytime and anywhere.
+        E-learning is nothing but the use of technology to connect.
+        teachers and students who are physically miles apart.
+        Learning objectives are the cornerstone of every e-learning course. They’re the reason you’re
+        creating the course. They guide you as you select the content and activities to include.
+        And they help you determine whether your course has been effective.
+        So basically, without learning objectives you won’t know why you’re creating the
+        course, what content to include, what activities to choose, or how to gauge the success of your course.
+        Seems pretty important when you put it that way, doesn’t it?
+      </div>
+      <div class="contact">
+        <p>
+          Please contact us per Email for any further questions about E-learning .
+        </p>
+        <p>
+          <strong><a href="ibouaik@gmail.com">ibouaik@gmail.com</a></strong>
+         </p>
+      </div>
+    </section>
+    <section class="about-logo">
+      <div class="title">
+        <h2>E-learning and 2022 Logo</h2>
+      </div>
+      <div class="concern">
+        <p>
+          The logo of E-learning and learning will be decided through the logo
+          competiton form 8 June to 7 July. After online public voting and
+          Selection Committee's judging process. the logo that will win prize.
+          After consultation with Mr. Agrawal will be optimized the
+          prize-winning-logo for the coming years.
+        </p>
+        <img src="./images/elearning-logo.png" alt="logo" />
+      </div>
+    </section>
+    <section class="past-activities">
+      <h2 class="activ-title">See the past GC campain Summites</h2>
+      <p>
+        Take a look at the last two GC Campain which took place in the last
+        year in Idia and Moroccoo
+      </p>
+      <div class="activities-details">
+        <div class="activity1">
+          <div>
+            <span class="date">Mai</span>
+            <span class="place">GC Campain in India</span>
+          </div>
+        </div>
+        <div class="activity2">
+          <div>
+            <span class="date">September</span>
+            <span>GC Campain in Morocco</span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="about-last">
+      <div class="about-last-container">
+        <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
+        <div>
+          <p>&copy; 2022 creative-commons-license-logo. Some Rights Reserved.</p>
+          <p>Created by Ilham Bouaik</p>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly try to match the Behance design in the `footer` section on the about page (same footer as on the home page) 
- [ ] Please make the footer light in the mobile version and dark in the desktop version as in the Behance design.
- mobile version:
![image](https://user-images.githubusercontent.com/24703171/164566283-3d342787-9e21-4d6f-8ddf-b647d64cbbd4.png)
- desktop version:
![image](https://user-images.githubusercontent.com/24703171/164566333-aec9f087-a6b2-49d6-8ac3-edefa6b40480.png)

- [ ] Kindly change the `div` tag to be `footer` tag for semantic purposes.